website/docs/language/settings/backends/configuration.mdx

```diff
@@ -156,6 +156,17 @@ naming pattern. Terraform will not prevent you from using other names but follow
 this convention will help your editor understand the content and likely provide
 better editing experience as a result.
 
+- Using a backend configuration file  or CLI key-value pairs overrides the contents, 
+but not the type of `backend`. (e.g. Use a Partial Configuration in the primary config.)
```

Comment:
```suggestion
You can use the `-backend-config` flag on the CLI to provide a backend configuration. You can either specify a file containing a partial configuration or specify a set of key-value pairs. Terraform applies the configurations to the arguments in the `backend` block, but it does not change the type.  In the following example, Terraform stores state in the default backend using the arguments supplied in the `-backend-config` flag:
```
I think this is the point you are making?

Reply:
Yes but it does miss one of the key things I wanted to document. I was surprised when trying to use this that I needed a partial configuration in the primary config, I guessed I could have just provided the backend from scratch in the override.